packages/grafana-toolkit/README.md

```diff
@@ -1,55 +1,82 @@
-# Grafana Toolkit
 
-Make sure to run `yarn install` before trying anything!  Otherwise you may see unknown command grafana-toolkit and spend a while tracking that down.
+# grafana-toolkikt
+grafana-toolkit is a simple CLI that enables efficient Grafana extensions development
 
+## Why grafana-toolkit?
+Historically, creating Grafana extension was an excercise of reverse engineering and ceremeny around testing, developing and eventually building the plugin. We want to help our community to focus on the core value of their plugins rather than all the setup required to develop an extension.
 
+With grafana-toolkit we put a simple CLI in your hands that address common tasks performed when developing an extension:
```

Comment:
Here, we're starting to explain more about the tool itself (rather than it's motivation). This (and following sections) probably belong under a "Usage" header.

Also, before going into how to use it, it's probably a good idea to provide instructions ("Install/Installation") on how to install the CLI. I assume you run something like `npm install -g @grafana/toolkit`?